app/uk/gov/hmrc/organisationsmatchingapi/services/MatchingService.scala

```diff
@@ -1,45 +0,0 @@
-/*
```

Comment:
I assume we are deleting this such that we can write our own implementation later?

Reply:
Yes - it was just duplicating the CacheService, it didn't have any implementation in it